tgb/utils/utils.py

```diff
@@ -12,6 +12,18 @@
 import torch
 
 
+_VERBOSE = (os.getenv("TGB_VERBOSE", 'False') == 'True') #only allows TGB_VERBOSE=True to set _VERBOSE to true

```

Comment:
Consider making this a bit more leniant
```suggestion
_VERBOSE = os.getenv("TGB_VERBOSE", 'False').lower() in ['true', '1']
```